### PR TITLE
fix(ui): `UserAvatarTransform.userAvatarBuilder` works only for other user.

### PR DESCRIPTION
**Pull request by xsahil03x**
Fixes: #1476

### CODE REVIEW

File: packages/stream_chat_flutter/CHANGELOG.md [hunk 1]
```diff
@@ -6,27 +6,34 @@
   Consonants as Emoji.
 - [#1505](https://github.com/GetStream/stream-chat-flutter/issues/1505) Fixed Message bubble disappears for Hangul
   Consonants.
+- [[#1476]](https://github.com/GetStream/stream-chat-flutter/issues/1476) Fixed `UserAvatarTransform.userAvatarBuilder`
```
Comment (kanat) [inline packages/stream_chat_flutter/CHANGELOG.md:9]: Is there any purpose for double square brackets? 🤔

Comment (xsahil03x) [reply packages/stream_chat_flutter/CHANGELOG.md:9]: It renders with bracket.

Eg: [#1426]

File: packages/stream_chat_flutter/CHANGELOG.md [hunk 2]
```diff
@@ -36,17 +43,22 @@
 - Updated `photo_manager` dependency to `^2.5.2`
 
 🐞 Fixed
-- [[#1424]](https://github.com/GetStream/stream-chat-flutter/issues/1424) Fixed a render issue when showing messages starting with 4 whitespaces.
+
+- [[#1424]](https://github.com/GetStream/stream-chat-flutter/issues/1424) Fixed a render issue when showing messages
```
Comment (kanat) [inline packages/stream_chat_flutter/CHANGELOG.md:47]: same here

**Review by kanat — DISMISSED** [review]
LGTM

**Pull request merged**